scripts/data/src/org/akvo/gae/remoteapi/FixOrphanedQuestionAnswers.java

```diff
@@ -0,0 +1,107 @@
+package org.akvo.gae.remoteapi;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.FetchOptions;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.repackaged.org.apache.commons.io.FileUtils;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import static org.akvo.gae.remoteapi.DataUtils.batchDelete;
+
+public class FixOrphanedQuestionAnswers implements Process {
+
+    @Override
+    public void execute(DatastoreService ds, String[] args) throws Exception {
+        String formId = args[0];
+        System.out.printf("Will do cleanup for form %s\n", formId);
+
+        long formId1 = Long.parseLong(formId);
+        ds.delete(KeyFactory.createKey("QuestionAnswerStore", 119950001));
```

Comment:
why do we have this hard coded id 119950001?

Reply:
ups... will fix